Add note under prerequisites on architecture (#8954)
grafana/loki: docs/sources/getting-started/_index.md

- [Docker](https://docs.docker.com/install)
- [Docker Compose](https://docs.docker.com/compose/install)

## Note

> The instruction only cover for system running on x86_64 architecture.

Reviewer: ```suggestion
These instructions are for a system running on an x86_64 architecture.
```

> It is not designed for arm/arm64 architecture.

Reviewer: ```suggestion
It is not designed for an arm/arm64 architecture.
```
